src/logic/export/RectLabelsExporter.ts

```diff
@@ -88,13 +85,10 @@ export class RectLabelsExporter {
                 }
             });
 
-        const projectName: string = EditorSelector.getProjectName();
-        const date: string = moment().format('YYYYMMDDhhmmss');
-
         try {
             zip.generateAsync({type:"blob"})
                 .then(function(content) {
-                    saveAs(content, `labels_${projectName}_${date}.zip`);
+                    saveAs(content, `${ExporterUtil.getExportFileName()}.zip`);
```

Comment:
bardziej mi chodizło o coś w rodzaju takiego obiektu, z którego można złożyć ten filename.
masz zip, csv, (chyba jeszcze json?)
więc taki obiekt w rodzaju:
```
{
    extension: `json`, // csv, zip
    template: 'label_${__PROJECTNAME__}_${__DATE__}',
}
```

to jest bardzo roboczy przykład :P

Reply:
i z takiego obiektu budować filename, np:
ExporterUtil.getFilename(object)

Reply:
ale w sumie przypierdzielam się. jak nie chcesz to nie rób :P